src/components/CategoryCard.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Typography } from "@mui/material";
+
+type CategoryCardProps = {
+  categoryName: string;
+  categoryImage: string;
+};
+
+const CategoryCard = ({ categoryName, categoryImage }: CategoryCardProps) => {
```

Comment:
React.FC<CategoryCardProps>,


decide are you using types ir interfaces for props